SWBF2Admin/Runtime/Commands/Permissions/CmdRmGroup.cs

```diff
@@ -61,7 +61,7 @@ public override bool AffectPlayer(Player affectedPlayer, Player player, string c
                 return false;
             }
 
-            Core.Database.RemovePlayerGroup(player, group);
+            Core.Database.RemovePlayerGroup(affectedPlayer, group);
```

Comment:
This was just fixing a bug where it would remove the caller from the group, not the target